src/reference/asciidoc/logging.adoc

```diff
@@ -270,6 +270,42 @@ public class MyEnhancedAppender extends AmqpAppender {
 ----
 ====
 
+Starting with <BACKLOG VERSION>, the log4j2 `AmqpAppender` can be extended using `@PluginBuilderFactory` and extending also
+`AmqpAppender.Builder`
+
+
+.log4j2
+====
+[source, java]
+----
+public class MyEnhancedAppender extends AmqpAppender {
```

Comment:
I think such a sample would be great candidate to have in our tests. 
Also consider to show how to use it in the config - I believe the test is not going to work without a proper logging config.

Reply:
I have done it in the next commit

Reply:
There is no need in making excuses - just push a commit.
Please, consider to add comments into PR only if you are not agree with me or I really ask about some clarification.
If we are on the same page, you just push commit, I get a (single!) notification and we are good to go.
Right now I have at about 10 emails from you and only with an info that you are going to fix it.

Now imaging that I have like at about 10 PRs in parallel and everybody is going to ask excuse for each my comment...
I feel so counterproductive! 😄 

Thank you for understanding!

Reply:
I'm not sure why you have resolved this conversation, but I also asked about an XML config sample.
And I with the test I understand what to do - the `@Plugin` is the trick! But you don't show it in this sample for the doc.
Or do you mean mean that `@Plugin` is optional and the one on the super class has an effect?

Reply:
Annotation `@Plugin`  will not be inherited and is not optional. Added into the doc